src/t8_geometry/t8_geometry_implementations/t8_geometry_linear_axis_aligned.cxx

```diff
@@ -82,6 +104,13 @@ t8_geometry_linear_axis_aligned::t8_geom_point_batch_inside_element (t8_forest_t
   return;
 }
 
+bool
+t8_geometry_linear_axis_aligned::t8_geom_tree_negative_volume (const t8_cmesh_t cmesh) const
+{
+  T8_ASSERT (correct_point_order (active_tree_vertices));
```

Comment:
Shouldn't we just return true, if the vertices are not in the correct order? Because this would lead to a negative volume, right?

Reply:
No, because it is a requirement for this geometry that the vertices are ordered correctly. Especially for the is_inside check. Therefore, we use an assertion instead of an if/else.